lib/src/git.rs

```diff
@@ -201,10 +201,12 @@ impl GitImportError {
 }
 
 /// Describes changes made by `import_refs()` or `fetch()`.
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq, Default)]
 pub struct GitImportStats {
     /// Commits superseded by newly imported commits.
     pub abandoned_commits: Vec<CommitId>,
+    /// Newly fetched remote refs.
+    pub new_remote_refs: BTreeSet<RefName>,
```

Comment:
nit: rename to `changed_remote_refs`? "new" might be confusing because the set contains created/deleted/moved refs.

Reply:
resolved